packages/sst/src/constructs/SsrSite.ts

```diff
@@ -283,6 +283,7 @@ export interface SsrSiteProps {
       | "logRetention"
     >;
   };
+  enableIAMAuth?: boolean;
```

Comment:
let's add some jsdoc on this new property

Reply:
can you add the default value as well ? this will automatically create documentation on SST website